Resources.it.resx

```diff
@@ -0,0 +1,1044 @@
+  </data>
+  <data name="SettingsAutoBackgroundTooltip" xml:space="preserve">
+    <value>Imposta il colore di sfondo sulle impostazioni del colore di Windows.</value>
+    <comment>Settings auto background tooltip</comment>
+  </data>
+  <data name="SettingsAutoUpdate" xml:space="preserve">
+    <value>Aggiornamento automatico</value>
+    <comment>Settings auto update label</comment>
+  </data>
+  <data name="SettingsAutoUpdateTooltip" xml:space="preserve">
+    <value>Aggiorna l'app automaticamente.</value>
+    <comment>Settings auto update tooltip</comment>
+  </data>
+  <data name="SettingsBackgroundColor" xml:space="preserve">
+    <value>Colore di sfondo</value>
+    <comment>Settings background color label</comment>
+  </data>
+  <data name="SettingsBackgroundColorTooltip" xml:space="preserve">
+    <value>Colore dello sfondo.</value>
+    <comment>Settings background color tooltip</comment>
+  </data>
+  <data name="SettingsBackgroundOpacity" xml:space="preserve">
+    <value>Opacità di sfondo</value>
+    <comment>Settings background opacity label</comment>
+  </data>
+  <data name="SettingsBackgroundOpacityTooltip" xml:space="preserve">
+    <value>Opacità dello sfondo.</value>
+    <comment>Settings background opacity tooltip</comment>
+  </data>
+  <data name="SettingsBandwidthInAlert" xml:space="preserve">
+    <value>Allerta della larghezza di banda</value>
+    <comment>Settings bandwidth in alert label</comment>
+  </data>
+  <data name="SettingsBandwidthInAlertTooltip" xml:space="preserve">
+    <value>La soglia kbps o kBps alla quale si verificano gli avvisi di larghezza di banda ricevuti. Usa 0 per disabilitare.</value>
+    <comment>Settings bandwidth in alert tooltip</comment>
+  </data>
+  <data name="SettingsBandwidthOutAlert" xml:space="preserve">
+    <value>Avviso di larghezza di banda</value>
+    <comment>Settings bandwidth out alert label</comment>
+  </data>
+  <data name="SettingsBandwidthOutAlertTooltip" xml:space="preserve">
+    <value>Soglia kbps o kBps alla quale si verificano gli avvisi di larghezza di banda inviati. Usa 0 per disabilitare.</value>
+    <comment>Settings bandwidth out alert tooltip</comment>
+  </data>
+  <data name="SettingsClickThrough" xml:space="preserve">
+    <value>Clicca attraverso</value>
+    <comment>Settings click through label</comment>
+  </data>
+  <data name="SettingsClickThroughTooltip" xml:space="preserve">
+    <value>Fa passare gli eventi del mouse attraverso la finestra. Nota che ciò impedirà qualsiasi interazione con la barra laterale.</value>
+    <comment>Settings click through tooltip</comment>
+  </data>
+  <data name="SettingsCollapseMenuBar" xml:space="preserve">
+    <value>Comprimi barra dei menu</value>
+    <comment>Settings collapse menu bar label</comment>
+  </data>
+  <data name="SettingsCollapseMenuBarTooltip" xml:space="preserve">
+    <value>Rimuove lo spazio aggiuntivo nella parte superiore della barra laterale.</value>

```

Comment:
Rimuovi gli spazi nella parte alta della barra laterale

Reply:
If this is an explanation, then I would do the following: 'Rimuove gli spazi nella parte alta della barra laterale'.
Using a generic 'spaces' (gli spazi) will work for future adjustments.

Reply:
Yes it works.

Reply:
@Pols88 please update the PR to use value: `Rimuove gli spazi nella parte alta della barra laterale`
Thank you